src/lib/FlightTasks/tasks/ManualPositionSmooth/FlightTaskManualPositionSmooth.cpp

```diff
@@ -57,7 +57,7 @@ void FlightTaskManualPositionSmooth::_updateSetpoints()
 	_velocity_setpoint(0) = vel_sp_xy(0);
 	_velocity_setpoint(1) = vel_sp_xy(1);
 
-	/* Check for altitude lock.*/
+	/* Check for xy position lock.*/
```

Comment:
I think @MaEtUgR already told you that we agreed on to use `//` for simple comments and `/**<` etc as the doxygen comments. It is not consistent yet mainly for the older Flighttasks